Cargo.toml

```diff
@@ -9,6 +9,7 @@ authors = ["Viam, inc. <contact@viam.com>"]
 
 [lib]
 crate-type = ["cdylib","lib"]
+doctest = false
```

Comment:
I started running `cargo test --workspace` instead of `cargo test --test "*"` in `etc/test.sh`. We now run all tests in the workspace instead of just all available from the top level directory. That includes doc tests, which for some reason fail to even compile. I can look more into that if you want, but for now I've just disabled doctests in the library.

Reply:
Seems reasonable to me, maybe we can make a ticket to revisit the doctest issue?

Reply:
Yep! https://viam.atlassian.net/browse/RSDK-4119.